apps/rush-lib/src/schemas/command-line.schema.json

```diff
@@ -80,6 +80,11 @@
               "title": "Allow Warnings in Successful Build",
               "description": "By default, Rush returns a nonzero exit code if errors or warnings occur during build. If this option is set to \"true\", Rush will return a zero exit code if warnings occur.",
               "type": "boolean"
+            },
+            "watchForChanges": {
```

Comment:
You also need to update [assets/rush-init/common/config/rush/command-line.json](https://github.com/microsoft/rushstack/blob/master/apps/rush-lib/assets/rush-init/common/config/rush/command-line.json) with these docs.